pkg/client/client.go

```diff
@@ -283,6 +284,13 @@ func New(c *restclient.Config) (*Client, error) {
 	return &Client{client}, nil
 }
 
+// DiscoveryClient returns a discovery client.
+func (c *Client) Discovery() discovery.DiscoveryInterface {
+	d := discovery.NewDiscoveryClient(c.RESTClient)
```

Comment:
If we're using this, want to remove the cruft from `pkg/client/discovery.go` where we combine results from `/api` and `/oapi` "for the short-term/forever"?